lumicks/pylake/detail/utilities.py

```diff
@@ -1,3 +1,7 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import colors
```

Comment:
Prefer `import matplotlib as mpl` and then `mpl.colors`.

Reply:
👍